core/templates/dev/head/domain/subtopic_viewer/subtopic-viewer-backend-api.service.ts

```diff
@@ -16,62 +16,60 @@
  * @fileoverview Service to get subtopic data.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
-require('domain/subtopic_viewer/subtopic-viewer-domain.constants.ajs.ts');
```

Comment:
I think it is because of this change. Now the .ajs.ts file isn't imported, which would mean that when the tests for this file run, the .ajs.ts file won't be included, and possibly it doesn't appear in the coverage report..

Ideally, wouldn't the .ajs.ts file be deleted in this PR? Or is it used elsewhere? If you are deleting it here, you'll need to remove the file name from the whitelist in scripts/check_frontend_coverage.py. If not, then as it was covered in the tests before, the PR should cover those lines I think.

Reply:
Yes I think I get what you're saying. 

Let me know if I'm right. So, in the whitelist field we've this file listed which means coverage for this file should be 100%. Since we aren't using this file, the lines for these file are not covered under any test and that's leading to failing frontend tests.